src/components/Header.js

```diff
@@ -1,16 +1,17 @@
 import React from 'react';
-import { Container } from 'reactstrap';
 import g from 'glamorous';
+import { rhythm } from '../utils/typography';
 import Menu from './Menu';
-
-const NavbarWrapper = g.div(props => ({
-  backgroundColor: props.theme.brand,
-}));
+import RibbonContainer from './RibbonContainer';
 
 export default () => (
-  <NavbarWrapper>
-    <Container>
+  <RibbonContainer
+    color="white"
+    backgroundColor="#E25E3E"
```

Comment:
We should wrap this component in `withTheme` and use `props.theme.brand` for this color instead of hard coding.

Reply:
aye